doc-content/enterprise-only/openshift/hacep-server-maven-proc.adoc

```diff
@@ -0,0 +1,132 @@
+[id='hacep-server-maven-proc']
+= Implementing the HA CEP server with a Maven repository for updating the KJAR service
+
+You can implement the HA CEP server that retrieves the KJAR service and all dependencies from a Maven repository that you provide. In this case, you can update the KJAR service at any time by updating it in the Maven repository and then making a call from the client code.
+
+You must prepare the source, build it, and then deploy it on {OPENSHIFT}. You must set certain environment variables in the `deployment.yaml` file before deploying the server. To use a Maven repository, you must set the `UPDATABLEKJAR` variable to `true`.
```

Comment:
Prepare the source, build it, and then deploy it on {OPENSHIFT}. Set the environment variables in the `deployment.yaml` file before deploying the server. To use a Maven repository, you must set the `UPDATABLEKJAR` variable to `true`.